Backend/api/user/controller.js

```diff
@@ -19,7 +21,6 @@ const getUser = async (req, res, next) => {
     }
 
     user["password"] = undefined;
```

Comment:
This is error prone. If someone performs a user query somewhere else and forgets to remove this field, the password hashes are leaked to clients. 

This can be moved in the `toJSON()` method of the schema, see the link below.
https://stackoverflow.com/questions/32390996/remove-one-property-from-mongoose-tojson-support/32391487